cider-dap/calyxDebug/cider-dap

```diff
@@ -0,0 +1 @@
+/home/basantkhalil/calyx2/target/debug/cider-dap
```

Comment:
This is a symlink and should not be committed. Ideally we should configure the
extension to use a local path to the bulid directory but for the time being a
symlink is acceptable but should not be commited since it's local to your file system